src/App.js

```diff
@@ -788,18 +789,39 @@ export default class App extends Component {
     const response = await fetch(endpoint);
     const body = await response.json();
     const {
-      proposals: proposalsList,
+      proposals,
       voteEndTime,
-      voteStartTime
+      voteStartTime,
+      trashAddress
     } = body.contents;
+
+    const proposalsList = proposals.map((proposal, i)=>{
+      return {
+        ...proposal,
+        _id: i
+      }
+    });
+
+    console.log({proposalsList})
+
+    const proposalsDictionary = proposalsList.reduce((acc,item)=>{
+      const {_id} = item;
```

Comment:
You really like destructuring assignment :-) Don't see much sense in it here though

Reply:
I do, actually :)
I also prefer consistent code over minimalistic one. 
It's possible to convert it into one liner, but it doesn't really matter that much, since it's still will be obfuscated and minimized and work exactly the same ^__^

Reply:
Can we mark this one as resolved or still want me to change it? :)